Makefile

```diff
@@ -79,6 +79,8 @@ PACKAGES := $(addprefix packages/,$(PKGNAMELIST))
 
 $(foreach pkg,$(PKGLIST),$(eval $(call build-package,$(pkg))))
 
+$(eval $(call build-package,cue,0.4.3-r0))
```

Comment:
We're in the process of changing our build process to not require these updates to the makefile, and instead we'll update `packages.txt`. Can you make that change there so we can build and merge this?

Reply:
thanks, didn't know that, fixed.